### DISCUSSION TITLE
Optimize command - extend which commands to execute

### DISCUSSION THREAD

**Discussion by pionl** [laravel/framework#43729] [Ideas]
Hi,

I have few "cache" commands that are used in production. For easy package extendability it would be great if we could "register" commands that optimize (clear too) will trigger.

Is this what Laravel would support? I can make the contribution.

Thanks for the info,
Martin

**Comment by mihai-stancu** [comment]
@pionl did you get ahead on this?

I'm guessing extending optimize locally isn't that hard but it'd be great to be able to register commands to optimize.

**Reply by pionl** [reply]
I was mainly waiting for a response (maybe from Laravel team). I would love to do it but don't won't to waste my time if Laravel team does not like it :)
